Content.Server/Cloning/CloningSystem.cs

```diff
@@ -306,6 +327,12 @@ private void EndFailedCloning(EntityUid uid, CloningPodComponent clonePod)
 
             var tileMix = _atmosphereSystem.GetTileMixture(transform.GridUid, null, indices, true);
 
+            if (HasComp<EmaggedComponent>(uid))
+            {
+                _audio.PlayPvs(clonePod.ScreamSound, uid, AudioParams.Default.WithVolume(4));
+                Spawn(clonePod.MobSpawnId, Transform(uid).Coordinates);
+            }
```

Comment:
```suggestion
            if (HasComp<EmaggedComponent>(uid))
            {
                _audio.PlayPvs(clonePod.ScreamSound, uid);
                Spawn(clonePod.MobSpawnId, transform.Coordinates);
            }
```

As above put the params on the audio itself.